scripts/player.py

```diff
@@ -78,9 +77,8 @@ def _is_action_correct(self, decision, index):
 
 
 class RealPlayer():
-    def __init__(self, path):
-        self.d = ExcelData(path=path)
-        self.data = self.d.prepare_data()
+    def __init__(self, data):
+        self.data = data
```

Comment:
let's just pass path as parameter, we dont really need all those fields we can only leave line 86